src/RpcClient.ts

```diff
@@ -207,7 +235,7 @@ export class RedirectRpcClient extends RpcClient {
         const message = UrlRpcEncoder.receiveRedirectResponse(window.location);
         if (message) {
             this._receive(message);
-        } else {
+        } else if (!UrlRpcEncoder.receiveRedirectCommand(window.location)) {
```

Comment:
Can you add a comment explaining this change?
I guess it's meant for apps like the accounts manager that are a RPC client and server at the same time.
Without that knowledge, it's not obvious why a `RpcClient` checks for a command which is meant for a `RpcServer`.